### ISSUE TITLE
XML access expressions are not defined in the 2019R3 Spec

### ISSUE THREAD

**Issue by hasithaa** [ballerina-platform/ballerina-lang#18875]
**Description:**

Following xml operations are supported.
```ballerina
import ballerina/io;public function main() {
    xml bookXML = xml `<book>
                <name>Sherlock Holmes</name>
                <author>
                    <fname title="Sir">Arthur</fname>
                    <mname>Conan</mname>
                    <lname>Doyle</lname>
                </author>
                <!--Price: $10-->
                </book>`;
    
    io:println(bookXML.author.fname);
    io:println(bookXML["author"]["fname"]);
    io:println(bookXML.ISBN.code);
    io:println(bookXML["ISBN"]["code"]);
    io:println(bookXML.author.fname.getTextValue());
    io:println(bookXML["author"]["fname"].getTextValue());
    io:println(bookXML.author.fname@["title"]);
    io:println(bookXML["author"]["fname"]@["title"]);
}
```

**Comment by hasithaa** [comment]
Related to https://github.com/ballerina-platform/ballerina-lang/issues/19570

**Comment by rdhananjaya** [comment]
Fixed in 1.2 and https://github.com/ballerina-platform/ballerina-lang/pull/22106
